Tests/BootCamp.Chapter.Tests/Expected/CityItemsMax.csv

```diff
@@ -1 +1 @@
-﻿Springfield, London
\ No newline at end of file
+﻿Springfield
```

Comment:
Modified test data file now should only output Springfield

Reply:
Yup.